Current_SamplingFunctionality.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <cstring>
+#include "Current_SamplingFunctionality.hpp"
+using namespace std;
+
+
+unsigned int ConsecutiveRanges(int Current_Samples[], int Current_Samples_size) {
+
+    unsigned int count = 1;
```

Comment:
What does it count? Avoid generic names like `count` and express the purpose in the name.